Makefile

```diff
@@ -145,12 +145,12 @@ else
 endif
 
 _do_uninstall:
-# It's safer to delete all workspaces before deleting the controller; otherwise we could
-# leave workspaces in a hanging state if we add finalizers.
+	# It's safer to delete all workspaces before deleting the controller; otherwise we could
+	# leave workspaces in a hanging state if we add finalizers.
```

Comment:
Indented comments are printed when executing the rule.

Reply:
I'll revert that. It was done to suppress the warning 
![Screenshot_20200721_174526](https://user-images.githubusercontent.com/5887312/88069326-fdeefa00-cb79-11ea-8019-b2be128c32c9.png)
But it seems to be like a bug on editor